src/software/estop/threaded_estop_reader.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#pragma once
+
+
+#include <boost/asio.hpp>
+#include <iostream>
+#include <mutex>
+#include <thread>
+
+#include "software/util/make_enum/make_enum.h"
+#include "uart_communication.h"
+
+// enum that represents the possible states of estop
+MAKE_ENUM(EstopState, PLAY, STOP, STATUS_ERROR);
```

Comment:
i think its still useful to keep, since the logic we perform when we receive an unknown message is different from a STOP message. Also allows for more informative logs.